src/components/inputs/CommentInput.tsx

```diff
@@ -77,6 +80,43 @@ export const CommentInput = ({ createComment, task_id }: Prop) => {
     }
   }, [detail, isListOrMenuActive]) // Depend on detail to ensure the latest state is captured
 
+  const uploadFn = token
+    ? async (file: File) => {
+        if (activeTask) {
+          const fileUrl = await uploadImageHandler(file, token ?? '', activeTask.workspaceId, task_id)
+          return fileUrl
+        }
+      }
+    : undefined
```

Comment:
```suggestion
  const uploadFn = token && async (file: File) => {
        if (activeTask) {
          const fileUrl = await uploadImageHandler(file, token ?? '', activeTask.workspaceId, task_id)
          return fileUrl
        }
      }
```
This should work if token is string | undefined

Reply:
I dont think this is possible because this is passed directly to tapwrite prop `uploadFn` which expects something like `(file : File) => promise('string' | undefined')` and not `(" " | (file : File) => promise('string' | undefined'))`